sopel/irc/__init__.py

```diff
@@ -309,6 +309,7 @@ def on_message(self, message: str) -> None:
             message,
             url_schemes=self.settings.core.auto_url_schemes,
             identifier_factory=self.make_identifier,
+            statusmsg_prefixes=self._isupport.get('STATUSMSG'),
```

Comment:
Why the `_` version?

```suggestion
            statusmsg_prefixes=self.isupport.get('STATUSMSG'),
```

Reply:
Prevent a call to a property which create one more call than necessary (object -> property -> _ version).